dev-docs/design/ssr-react.md

```diff
@@ -0,0 +1,72 @@
+# Server Rendering Lit Components in React
+
+## Background
+
+React server rendering of custom elements will only shallow render components by emitting the host element tag with props stringified to attributes and any light DOM children it has. `@lit-labs/ssr` is able to server render Lit elements deeply by emitting the contents of the component as Declarative Shadow DOM. We want usage of custom elements in React with server rendering to be able to take advantage of this as well.
+
+Client rendering of custom elements in React have their shortfalls such as all props being set as attributes on the element and adding event listeners being clunky. We have the `createComponent` in `@lit-labs/react` that creates a React Component that wraps the custom element which helps with this situation.
+
+## Scenarios to Handle
+
+<!-- prettier-ignore-start -->
+| | With vanilla React | What we want |
```

Comment:
Nit: Left side is "With vanilla React". Could right hand side be "With Lit SSR packages in React" or something more specific.

I found myself wondering - what do we want that is different from "With vanilla React".